content/en/docs/concepts/scheduling-eviction/dynamic-resource-allocation.md

```diff
@@ -200,11 +209,42 @@ spec:
 You may also be able to mutate the incoming Pod, at admission time, to unset
 the `.spec.nodeName` field and to use a node selector instead.
 
+## Admin access
+
+{{< feature-state feature_gate_name="DRAAdminAccess" >}}
+
+A ResourceClaim with admin access grants access to devices which are in use and
+may enable additional permissions when making the device available in a
+container:
+
+```yaml
+apiVersion: resource.k8s.io/v1beta1
+kind: ResourceClaimTemplate
+metadata:
+  name: large-black-cat-claim-template
+spec:
+  spec:
+    devices:
+      requests:
+      - name: req-0
+        deviceClassName: resource.example.com
+        adminAccess: true
+```
+
+If this feature is disabled, the `adminAccess` field will be removed
+automatically when creating such a ResourceClaim.
+
+Admin access is a privileged mode which should not be made available to normal
+users in a multi-tenant cluster. Cluster administrators can restrict usage of
+this features by installing a validating admission policy similar to
+[the in-tree example](https://github.com/kubernetes/kubernetes/blob/33ea278/test/e2e/dra/test-driver/deploy/example/admin-access-policy.yaml)
+when enabling this feature.
```

Comment:
```suggestion
Setting `adminAccess` true marks the allocated resources as privileged: facilities
or devices that should not be made available to normal
users in a multi-tenant cluster. As a cluster administrator, you can restrict usage of
privileged resources by installing a ValidatingAdmissionPolicy similar to
the [in-tree example](https://github.com/kubernetes/kubernetes/blob/33ea278/test/e2e/dra/test-driver/deploy/example/admin-access-policy.yaml)
when enabling this feature.
```

However, at beta we expect the example ValidatingAdmissionPolicy to be copied into the docs repo (possibly with a light tweak), rather than linking to code from k/k.

Reply:
> Setting `adminAccess` true marks the allocated resources as privileged: facilities or devices that should not be made available to normal users in a multi-tenant cluster.

This is not accurate. Already the request is special (scheduling) and then the access to them in containers *for this request*. Elsewhere, the devices are normal and can be used by normal users.

Reply:
> However, at beta we expect the example ValidatingAdmissionPolicy to be copied into the docs repo (possibly with a light tweak), rather than linking to code from k/k.

Inlined as a YAML code block, with tweaked documentation for admins.

Reply:
https://deploy-preview-48330--kubernetes-io-main-staging.netlify.app/docs/concepts/scheduling-eviction/dynamic-resource-allocation/#admin-access